### PR TITLE
Add sport event blog post

### PR DESCRIPTION
**Pull request by FriederikeZelke**

### CODE REVIEW

**Comment by berendt** [comment]
Filename should be ``2023-07-05Sportveranstaltung.md``.

**Comment by berendt** [comment]
Is there already an idea how big the individual teams are or will that only be decided when all the registrations are through?

**Comment by FriederikeZelke** [comment]
We plan teams with 3-4 members. We decide when all registrations are there.

File: _i18n/de/_posts/blog/2023-07-05Sortveranstaltung.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+### Regeln für die Veranstaltung:
+* In der Woche KW 36/37: 9. - 15.09.2023 absolviert jeder Teilnehmer:in seine/ihre Sportart und versucht in einem Rutsch an einem Tag so lange zu laufen/gehen/radeln/schwimmen, wie er/sie kann (es wird nur eine Strecke an einem Tag für das Endergebnis gewertet, die Teilnehmer:innen können aber natürlich an jedem Tag versuchen, ihre eigene Strecke zu schlagen)
+* Jede:r Teilnehmer:in meldet ihr/sein bestes Ergebnis, alle Ergebnisse eines Teams werden gesammelt, um das endgültige Teamergebnis zu erhalten.
+* Das Team mit den meisten Kilometern gewinnt
+* Wir werden einen Ausgleich für das Radfahren in den Bergen finden (Ideen hierzu sind sehr willkommen!)
+* E-Bikes oder virtuelles Radfahren/Laufen sind nicht erlaubt
+
+
+### Jeder Kilometer zählt - sponsern Sie unsere Sportveranstaltung
+Die Veranstaltung ist eine Sponsoren-/Spendenveranstaltung. Und es gibt mehrere Möglichkeiten zu sponsern:
```
Comment (maxwolfs) [inline _i18n/de/_posts/blog/2023-07-05Sortveranstaltung.md:26]:
```suggestion
Die Veranstaltung ist eine Sponsoren-/Spendenveranstaltung. Es gibt mehrere Möglichkeiten zu sponsern:
```

File: _i18n/de/_posts/blog/2023-07-05Sortveranstaltung.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* Sponsoren zahlen einen Betrag, den er selbst festlegt, für alle Sportarten und Teilnehmer für jeweils (x):
```
Comment (maxwolfs) [inline _i18n/de/_posts/blog/2023-07-05Sortveranstaltung.md:27]:
```suggestion
* Sponsoren zahlen einen Betrag, den sie selbst festlegen, für alle Sportarten und Teilnehmer für jeweils (x):
```

File: _i18n/de/_posts/blog/2023-07-05Sortveranstaltung.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Hoch mit dem Hintern!"
+author:
+  - "Friederike Zelke"
+avatar: 
+  - "Zelke.jpg"
+image: "blog/run.jpg"
+---
+{% asset 'blog/run.jpg' vips:format='.webp' style="width:100%; max-height: 350px; object-fit: cover;" %}
```
Comment (maxwolfs) [inline _i18n/de/_posts/blog/2023-07-05Sortveranstaltung.md:10]: Does this work? Within this PR is no committed `run.jpg`

**Review by maxwolfs — CHANGES_REQUESTED** [review]
I found some things, otherwise 👍

File: _i18n/de/_posts/blog/2023-07-05Sortveranstaltung.md (hunk 3 above)
Comment (FriederikeZelke) [reply _i18n/de/_posts/blog/2023-07-05Sortveranstaltung.md]: What do you mean?

**Review by FriederikeZelke — COMMENTED** [review]
LGTM

**Review by maxwolfs — APPROVED** [review]

**Pull request merged**